index.html

```diff
@@ -5,6 +5,6 @@
 
 <h1>This is a heading</h1>
 <p>This is a paragraph.</p>
-
+<h1>This is so much fun!</h1>
```

Comment:
https://www.youtube.com/watch?v=seICQOd2qsY, used same change title as the video provided.